lib/pbench/test/unit/server/test_datasets_compare.py

```diff
@@ -6,10 +6,16 @@
 import requests
 
 from pbench.server import JSON
-from pbench.server.cache_manager import CacheManager, Tarball
+from pbench.server.cache_manager import CacheManager
 from pbench.server.database.models.datasets import Dataset, DatasetNotFound, Metadata
 
 
+def mock_get_value(dataset, key) -> str:
+    if dataset.name == "uperf_3" or dataset.name == "uperf_4":
+        return "hammerDB"
```

Comment:
Nothing wrong here, it's a creative and convenient lie, but I can't help but be amused that two "uperf" datasets are now "hammerdb".  (And ... would the actual `dataset.metalog.pbench.script` value be "hammerdb" or "hammerDB" or something else? I wonder if we even have any... and again that's completely irrelevant since we just need them not to match. 😆 )